implementations.md

```diff
@@ -0,0 +1,19 @@
+# IPFS Implementations
+
+For a list of IPFS API Client Libraries, please refer to *[Client Libraries](https://github.org/ipfs/ipfs/client-libraries.md)*
+
+| Language | Project |
+|----------|---------|
+| Go* | https://github.com/ipfs/go-ipfs |
+| Javascript | https://github.com/ipfs/js-ipfs |
+| Python | https://github.com/ipfs/py-ipfs |
+| Haskell | https://github.com/cleichner/haskell-ipfs |
+| .net | https://github.com/wasabii/dotnet-ipfs |
+| C++ | https://github.com/subtly/cpp-ipfs |
+| Rust | https://github.com/rschulman/rust-ipfs |
+
+*reference
+
+Would you like to start your own language implementation of IPFS? Post an issue in this repository so we can help you. There is at present no "perfect spec" and many protocol details are still to be decided upon.
```

Comment:
Worth including a link to:  https://github.com/ipfs/specs/blob/master/overviews/implement-ipfs.md -- though nothing there yet, merging tomorrow: https://github.com/ipfs/specs/blob/overviews/overviews/implement-ipfs.md for now